MerchantSDK/GiniMerchantSDKExample/GiniMerchantSDKExample/Sources/InvoicesList/InvoiceDetailView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  InvoiceDetailView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+
+@MainActor
+class InvoiceDetailView: UIStackView {
+
+    static var textFields = [String: UITextField]()
+
+    convenience init(_ items: [(String, String)]) {
+        Self.textFields.removeAll()
+        self.init(arrangedSubviews: items.map { Self.view(for: $0) })
+
+        translatesAutoresizingMaskIntoConstraints = false
+        axis = .vertical
+        distribution = .fill
+        alignment = .fill
+        spacing = Constants.verticalSpacing
+    }
+
+    private class func view(for text: (String, String)) -> UIView {
+
+        let horizontalStackView = UIStackView()
+        horizontalStackView.translatesAutoresizingMaskIntoConstraints = false
+        horizontalStackView.axis = .horizontal
+        horizontalStackView.distribution = .fill
+        horizontalStackView.spacing = 0
+
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = text.0
```

Comment:
[Clarification]: Could you please help me to understand where we received the text and what's `text.0`?